comps/llms/summarization/tgi/langchain/llm.py

```diff
@@ -76,6 +93,26 @@ async def llm_generate(input: DocSumLLMParams):
     if logflag:
         logger.info(input)
 
+    ### check summary type
+    if input.summary_type not in ["auto", "stuff", "truncate", "map_reduce", "refine"]:
+        raise NotImplementedError(
+            'Please specify the summary_type in "auto", "stuff", "truncate", "map_reduce", "refine"'
+        )
```

Comment:
Better to specify long lists just once:
```suggestion
    summary_types = ["auto", "stuff", "truncate", "map_reduce", "refine"]
    if input.summary_type not in summary_types:
        raise NotImplementedError(
            f'Please specify the summary_type in {summary_types}'
        )
```